x-pack/test/accessibility/apps/transform.ts

```diff
@@ -112,6 +112,19 @@ export default function ({ getService }: FtrProviderContext) {
           );
           await transform.sourceSelection.selectSource(ecIndexPattern);
 
+          await transform.testExecution.logTestStep(
+            `sets the date picker to the default '15 minutes ago'`
+          );
+          await transform.datePicker.openSuperDatePicker();
+          await transform.datePicker.quickSelect(15, 'm');
+
+          await transform.testExecution.logTestStep('displays an empty index preview');
+          await transform.wizard.assertIndexPreviewEmpty();
+
+          await transform.testExecution.logTestStep(`sets the date picker to '10 Years ago'`);
+          await transform.datePicker.openSuperDatePicker();
+          await transform.datePicker.quickSelect();
```

Comment:
Is this meant to set to 10 years? The current default for `quickSelect` looks like 15 years.

Reply:
Updated the log text in [09f1e77](https://github.com/elastic/kibana/pull/149049/commits/09f1e77b7fe67ac603bfed2f00631458fb434f0b).